packages/mobile/src/navigator/DrawerNavigator.tsx

```diff
@@ -179,6 +181,8 @@ function CustomDrawerContent(props: DrawerContentComponentProps<DrawerContentOpt
 export default function DrawerNavigator() {
   const { t } = useTranslation()
   const isCeloEducationComplete = useSelector((state) => state.goldToken.educationCompleted)
+  const isDappsExplorerEnabled = useSelector((state) => state.app.dappsExplorerEnabled)
```

Comment:
To avoid having to handle the error in the other screen, can we check as well if the URI is available? And if it's not not show the item since we know it will fail :)

Reply:
Yep, I'll do that